api/client/options.proto

```diff
@@ -71,6 +71,20 @@ message SequencerIdleStrategy {
   SequencerIdleStrategyOptions value = 1;
 }
 
+message MultiTarget {
+  message Endpoint {
+    google.protobuf.StringValue address = 1;
+    google.protobuf.UInt32Value port = 2 [(validate.rules).uint32 = {gte: 1, lte: 65535}];
+  }
+  // Whether to use HTTPS in requests to all backends; otherwise HTTP.
+  google.protobuf.BoolValue use_https = 1;
+  // One or more address-port pairs to receive traffic distributed with round robin.
+  repeated Endpoint endpoints = 2;
+  // The absolute HTTP request path (the part of the URL after host:port).
+  // A single path is requested from all backends. Ignored when using a RequestSource.
```

Comment:
(nit) Might not be clear whether the path should or shouldn't start with a forward slash, can we include at least one example?

Reply:
Added an example